src/orange/detail/UnitInserter.cc

```diff
@@ -214,8 +214,7 @@ SimpleUnitId UnitInserter::operator()(UnitInput const& inp)
         });
 
     CELER_ASSERT(unit);
-    auto simple_units = make_builder(&orange_data_->simple_unit);
-    return simple_units.push_back(unit);
+    make_builder(&orange_data_->simple_units).push_back(unit);
```

Comment:
I think you should revert this (return the value) since we may end up using it. Returning the new ID doesn't hurt, anyway.